aws/resource_aws_ecr_repository_policy.go

```diff
@@ -78,7 +81,6 @@ func resourceAwsEcrRepositoryPolicyRead(d *schema.ResourceData, meta interface{}
 
 	log.Printf("[DEBUG] Reading repository policy %s", d.Id())
 	out, err := conn.GetRepositoryPolicy(&ecr.GetRepositoryPolicyInput{
-		RegistryId:     aws.String(d.Get("registry_id").(string)),
```

Comment:
I believe the intention here is that ECR may eventually support more than one registry per AWS account, but that is not currently supported nor does the `aws_ecr_repository` resource import require the leading registry ID (e.g. AWS account ID). So this seems okay.